packages/pg-v5/commands/pull.js

```diff
@@ -125,19 +112,69 @@ const maybeTunnel = function * (herokuDb) {
   return herokuDb
 }
 
+function spawnPipe (commandOne, commandTwo) {
+  return new Promise((resolve, reject) => {
+    commandOne.stdout.on('data', (data) => {
```

Comment:
can you do `commandOne.stdout.pipe(commandTwo.stdin)`? I think that would work anyways and is actually better

Reply:
@jdxcode unless I'm missing something, there is no function `pipe` that can be called on `stdout`.